### ISSUE TITLE
Update to Node 14+

### ISSUE THREAD

**Issue by danizen** [ExLibrisGroup/primo-explore-devenv#136]
The librarians I support want to be able to run primo-explore-devenv directly rather than relying on me all of the time.   The instructions say to use Node 10, but even Node 12 is almost end-of-life (see https://endoflife.date/nodejs).

For me, I can always install any version of Node, even if I have to hold my nose to do it.  My librarians will need approval from our Desktop Support IT section to install Node, and must install a currently maintained version.

I have experimentally determined that Node 14 and 16 do work, at least on MacBook Pro where "brew link node@16" is an easy change.

**Comment by ianchan** [comment]
Knowing you were able to make the newer versions work was most helpful. I was able to update to Node 14 but 16 didn't work for me. Node-sass in particular was problematic. I eventually had to install n to switch from 16 (installed via brew) to 14 and Xcode so I could install python2. After that, rebuilt node-sass. I'm not that familiar with Node so there's probably better ways. This was on a MacBook Pro with Apple M1.

**Comment by jlcolbert** [comment]
I figured out a way to update things, and also make them compatible with ES6. I need to do some maintenance on it, but that's the gist.

https://github.com/jlcolbert/primo-explore-devenv

**Comment by danizen** [comment]
That is very interesting - I speculate that since there will be AngularJS 1.8 soon that Ex Libris may try to update for ES6 and TypeScript to lay the ground work to update to Angular 4+.

**Comment by danizen** [comment]
I'm going to close this as we are satisfied with our current solution.

**Comment by danizen** [comment]
@ianchan, I cannot reproduce a successful build with Node 16.  I think I must have preserved `node_modules` sub-directory and then just run gulp run --view ... Now I also error out building node-sass.

**Comment by danizen** [comment]
At a recent webinar, another PrimoVE customer suggested nvm.  We are constrained to run Windows, so I looked at [nvm-windows](https://github.com/coreybutler/nvm-windows), but this will require administrative rights.  The solution is for ExLibris/Clarivate to support primo-explore-devenv better.

**Comment by Steve-A-Orr** [comment]
I agree with you all. There are many people (#138) who want this support from Ex Libris--how do we escalate the issue out of Github?

**Comment by gloriousgeneralist** [comment]
I'm escalating to Primo product management, also please join the Slack here and bring up those concerns https://join.slack.com/t/igelu-eluna-siwg/shared_invite/zt-1g2nga2vn-Of0i1ZVg77RYbgOJrOej8w (will expire in 7 days)

**Comment by ShoulM** [comment]
I am running the devenv successfully on Node v14 (v14.20.0). Am I missing something?

**Comment by gloriousgeneralist** [comment]
The instructions clearly say for installation Download and install the [Node version 10.15.3](https://nodejs.org/download/release/v10.15.3/)

This may be as simple as updating documentation, but would be good to know that it is confirmed to work.

**Comment by danizen** [comment]
@gloriousgeneralist, I don't think you understand the issue.  The desktop security will not allow librarians to install that version.  If Ex Libris is serious about supporting primo-explore-devenv, then they must support actual production node versions on an ongoing basis.

**Comment by gloriousgeneralist** [comment]
@danizen I understand the issue perfectly. My comment was directed at @ShoulM who is on the Ex Libris development team. I know someone mentioned bringing this up at IGELU and I'll also be following up again with Primo product management at our next meeting.

**Comment by ShoulM** [comment]
I opened a new branch that will support Node v16.17.0 LTS([Node-v16.17.0-(LTS)-support](https://github.com/ExLibrisGroup/primo-explore-devenv/tree/Node-v16.17.0-(LTS)-support)). So if you want to use the latest node LTS pull from that branch. 
I tested it on PC and it seems to working ok (just needed to upgrade node-sass dependency). 
If I don't get any issues with this branch I will merge it with the master branch in the future.
